store/tikv/region_cache.go

```diff
@@ -138,26 +168,24 @@ func (c *RegionCache) LocateKey(bo *Backoffer, key []byte) (*KeyLocation, error)
 
 // LocateRegionByID searches for the region with ID
 func (c *RegionCache) LocateRegionByID(bo *Backoffer, regionID uint64) (*KeyLocation, error) {
-	c.mu.RLock()
-	if r := c.getRegionByIDFromCache(regionID); r != nil {
+	r := c.getRegionByIDFromCache(regionID)
```

Comment:
@disksing Should we use `GetCachedRegion` here?
This method is only used by HTTP status server.

Reply:
They have different parameters. BTW, I think we need lock here. @atmzhou

Reply:
@disksing I remember that you said this function is only used for test. So I did not add lock in getRegionByIDFromCache.
Do you think we should add lock? If yes, I will add.

Reply:
Yes. We don't want debug API have data race too, especially it may affect online service.